README.md

```diff
@@ -89,6 +89,31 @@ You can also manually run `nosetests` with `coverage` (but `setup.cfg` does this
 Try and get as close to 100% coverage as you can.
 
 
+## Deploy to IBM Cloud manually
+The `manifest.yml` file must be edited with the configuration of the cloud where is application is to be deployed to.
+
+Add the `apiKey.json` file to a new directory `.bluemix` in the folder. This should enable logging into your IBM Cloud to deploy the application.
+
+Invoke Vagrant and log into IBM Cloud using the following commands.
+
+``` shell
+    $ vagrant up
+    $ ibmcloud login -a https://cloud.ibm.com --apikey @~/.bluemix/apiKey.json -r <region> -o <organization> -s <space> 
+``` 
+
+Ensure that the ElephantSQL URL is present in the `VCAP_SERVICES` environment variable of the runtime of the Cloud Foundry application.
```

Comment:
how would I do this?

Reply:
Already done for us, when you go to the [cloud.ibm.com](cloud.ibm.com) and open the Cloud Foundry app, go to ``` Runtime ```. Over there, a tab for ``` Environment variables ``` is present.

![image](https://user-images.githubusercontent.com/15267877/98167780-9dc95d00-1eb7-11eb-9ac9-b51f41e9cf3f.png)

Reply:
How do I get to our app in my account?